docusaurus/docs/publish-a-plugin/provide-provisioning.md

```diff
@@ -0,0 +1,60 @@
+---
+id: provide-provisioning
+title: Provision your plugin
+description: How to add provisioning to your plugin to potentially speed up your plugin review process.
+keywords:
+  - github
+  - provisioning
+  - provision
+  - review
+sidebar_position: 5
+---
+
+# Plugin provisioning
+
+Developers of community plugins often ask us these questions:
```

Comment:
This section comes across as unnecessarily negative to me and that we're just shifting the pain . 

Suggestion - (note the lazy links since I'm doing this in the web editor), may also need editing for more conversational tone as well.

>Developers often ask us how long it takes for a plugin to be reviewed for publishing to the Grafana `[plugin catalog](https://grafana.com/plugins)`. Whilst we `[can't provide estimates](link to the faq)` - we are always looking for ways to reduce cycle times, such as through additional automation.
>
>By far the most time consuming aspect of a plugin's review is the creation of a suitable test environment so we can verify its behaviour, often involving a number of back-and-forth conversations between the developer and the review team. To drastically improve the time it takes for a plugin to be reviewed, developers can provide this themselves through including the appropriate resources and configuration to be `[provisioned](https://grafana.com/docs/grafana/latest/administration/provisioning/#provision-grafana)` within the `[docker development environment](link to docker dev env)` we generate with the `create-plugin` tool.

Reply:
nice. edited a bit.